x-pack/test/apm_api_integration/tests/traces/trace_by_id.spec.ts

```diff
@@ -59,6 +59,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       `);
       expectSnapshot(
         response.body.traceDocs.map((doc) =>
+          // @ts-expect-error Property 'processor' does not exist on type 'Profile'.
```

Comment:
this (and the other @ts-expect-error before) indicate that the request body is not valid. I'll have a look.